google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITBlobReadChannelTest.java

```diff
@@ -54,26 +52,23 @@ public final class ITBlobReadChannelTest {
 
   @Rule public final TemporaryFolder tmp = new TemporaryFolder();
 
-  @Rule public final StorageFixture storageFixture = StorageFixture.defaultHttp();
+  @ClassRule(order = 1)
+  public static final StorageFixture storageFixture = StorageFixture.defaultHttp();
+
+  @ClassRule(order = 2)
+  public static final BucketFixture bucketFixture =
+      BucketFixture.newBuilder().setHandle(storageFixture::getInstance).build();
 
-  private Storage storage;
   private String bucketName;
   private String blobName;
 
   @Before
   public void setUp() throws Exception {
-    storage = storageFixture.getInstance();
```

Comment:
nit: I don't see the value of offloading this to all places that require an instance of storage.

Reply:
I can back this change out then.